pkg/tracing/tracing.go

```diff
@@ -18,9 +17,7 @@ import (
 	"k8s.io/client-go/kubernetes"
 )
 
-func ShutdownController(tp *sdktrace.TracerProvider, ctx context.Context, timeout time.Duration) {
-	ctx, cancel := context.WithTimeout(ctx, timeout)
-	defer cancel()
+func ShutDownController(tp *sdktrace.TracerProvider, ctx context.Context) {
```

Comment:
By convention, the `context.Context` is the first parameter.
```suggestion
func ShutDownController(ctx context.Context, tp *sdktrace.TracerProvider) {
```

Reply:
Ok, thanks for the review 👍